lib/profile/view_profile/header.dart

```diff
@@ -19,16 +20,27 @@ class ProfileHeader extends StatelessWidget {
           bottomLeft: Radius.circular(40.0)
         ),
       ),
-      child: Container(
-        padding: EdgeInsets.only(top: 30.0),
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget> [
-            buildProfilePhoto(context),
-            buildFullName()
-          ]
-        )
-      ),
+      child: Stack(
```

Comment:
I just curious why use Stack here? Is that better than Container?

Reply:
This is because I am using `Positioned` widgets here for the edit icon , user's full name and profile picture in the header. `Positioned` needs to be a descendant of `Stack`. You can read more about it here: https://api.flutter.dev/flutter/widgets/Positioned-class.html
In my opinion, unless you need to position widgets on top of each other, I would rather use `Container`.
You can also read more about `Stack` here and when it is better to use it: https://itnext.io/flutter-everything-about-stack-48fe809874e9